ietf-optical-impairment-topology.tree

```diff
@@ -49,39 +56,110 @@ module: ietf-optical-impairment-topology
                 +--:(concentratedloss)
                    +--ro concentratedloss
                       +--ro loss?   decimal64
-  augment /nw:networks/nw:network/nw:node/tet:te/tet:tunnel-termination-point:
+  augment /nw:networks/nw:network/nw:node/tet:te
+            /tet:tunnel-termination-point:
     +--ro OTSiG-element* [OTSiG-identifier]
     |  +--ro OTSiG-identifier    int16
     |  +--ro OTSiG-container
     |     +--ro OTSi* [OTSi-carrier-id]
     |        +--ro OTSi-carrier-id           int16
+    |        +--ro transponder-ref?          leafref
+    |        +--ro transceiver-ref?          leafref
+    |        +--ro configured-mode?          leafref
     |        +--ro OTSi-carrier-frequency?   decimal64
     |        +--ro OTSi-signal-width?        decimal64
     |        +--ro channel-delta-power?      decimal64
-    +--ro transponders-list* [transponder-id]
-       +--ro transponder-id                      uint32
-       +--ro (mode)?
-       |  +--:(G.692.2)
-       |  |  +--ro standard-mode?                standard-mode
-       |  +--:(organizational-mode)
-       |  |  +--ro operational-mode?             operational-mode
-       |  |  +--ro organization-identifier?      vendor-identifier
-       |  +--:(explicit-mode)
-       |     +--ro available-modulation-types*   identityref
-       |     +--ro configured-modulation-type?   identityref
-       |     +--ro available-baud-rates*         uint32
-       |     +--ro configured-baud-rate?         uint32
-       |     +--ro available-FEC-types*          identityref
-       |     +--ro configured-FEC-type?          identityref
-       |     +--ro FEC-code-rate?                decimal64
-       |     +--ro FEC-threshold?                decimal64
-       +--ro power?                              int32
-       +--ro power-min?                          int32
-       +--ro power-max?                          int32
-  augment /nw:networks/nw:network/nw:node/tet:te/tet:tunnel-termination-point:
-    +--ro transponder-list* [carrier-id]
+    |        +--ro central-frequency?        frequency-thz
+    |        +--ro channel-output-power?     dbm-t
+    |        +--ro channel-input-power?      dbm-t
+    |        +--ro total-input-power?        dbm-t
+    +--ro transponder-list* [transponder-id]
+       +--ro transponder-id      uint32
+       +--ro transceiver-list* [transceiver-id]
```

Comment:
Same comment as above about using the term 'list':

`       +--ro transceiver* [transceiver-id]
`

Reply:
ok. I will do for the next commit after review.